src/convertToObject.js

```diff
@@ -12,7 +12,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStyles = sourceString.split(':').join(';').split(';').map(item =>
```

Comment:
what's going on here? you join by ; to split again by ;?
maybe just split by ; at the beginning?